app/templates/reminders/new.hbs

```diff
@@ -0,0 +1,7 @@
+<form class="new-reminder-form" >
+  Title: {{input class="title-input" type="text" value=model.title placeholder="Title"}}
+  <br>
```

Comment:
Let's use CSS to add a bottom margin instead of `</br>` tags.  Looks like you already have a class on here so it shouldn't cause too much trouble :)